module/VuFind/src/VuFind/Db/Table/ResourceFactory.php

```diff
@@ -68,7 +68,10 @@ public function __invoke(
             throw new \Exception('Unexpected options sent to factory!');
         }
         $converter = $container->get(\VuFind\Date\Converter::class);
-        $loader = $container->get(\VuFind\Record\Loader::class);
-        return parent::__invoke($container, $requestedName, [$converter, $loader]);
+        // Wrapper needed to avoid circular dependency:
+        $populatorLoader = function () use ($container) {
```

Comment:
The database service currently depends on the table, and the populator depends on the database service, so we need to break the circle here. Once we migrate away from Laminas\Db, this will no longer be an issue.